packages/shared/src/components/fields/Dropdown.tsx

```diff
@@ -8,13 +8,15 @@ import React, {
 import classNames from 'classnames';
 import {
   Item,
-  ItemParams,
   Menu,
   TriggerEvent,
   useContextMenu,
 } from '@dailydotdev/react-contexify';
 import { ArrowIcon, VIcon } from '../icons';
 import styles from './Dropdown.module.css';
+import { useViewSize, ViewSize } from '../../hooks';
+import { ListDrawer } from '../drawers/ListDrawer';
+import { SelectParams } from '../drawers/common';
```

Comment:
Seeing this is only type remember we can use `import type { bla }` (not urgent, just incase you wanted to)

Reply:
Ah, yes, of course. I've been trying to use the most of it lately. Let me apply it here.